tests/testthat/test-utilities-enum.R

```diff
@@ -6,3 +6,16 @@ test_that("enums get/set", {
   expect_false(is.null(e))
   expect_that(e$call("ToString"), equals("B"))
 })
+
+test_that("instance enumerators are evaluated as int", {
```

Comment:
Get and Set evaluates whether the GetFieldOrProperty can use int types to interact with enums.